src/cli.ts

```diff
@@ -11,7 +11,7 @@ import { findAlreadyInitializable } from './find-already-initializable';
 
 async function getPaths() {
   const hardhat = require.resolve('hardhat', { paths: [process.cwd()] });
-  const hre: HardhatRuntimeEnvironment = await import(hardhat);
+  const hre: HardhatRuntimeEnvironment = (await import(hardhat)).default;
```

Comment:
Yes but it's more verbose. It would be like this:

```typescript
  const { default: hre }: { default: HardhatRuntimeEnvironment: = await import(hardhat);
```